docs/book/src/topics/ipv6.md

```diff
@@ -0,0 +1,104 @@
+# IPv6 clusters
+
+## Overview
+
+CAPZ enables you to create IPv6 Kubernetes clusters on Microsoft Azure.
+
+- IPv6 support is available for Kubernetes version 1.18.0 and later on Azure.
+- IPv6 support is in beta as of Kubernetes version 1.18 in Kubernetes community.
+
+To deploy a cluster using IPv6, create a cluster configuration with the [ipv6 flavor](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/master/templates/cluster-template-ipv6.yaml).
+
+:warning: Action required: The Azure DNS nameserver is only IPv4. If the coredns pod runs on the pod network, it will fail to resolve. 
```

Comment:
Confirmed that manually patching the coredns deployment is the only option  currently  https://kubernetes.slack.com/archives/C2P1JHS2E/p1599897213056200?thread_ts=1599861498.055900&cid=C2P1JHS2E